articles/quickstart/backend/python/01-authorization.md

```diff
@@ -10,7 +10,8 @@ description: This tutorial will show you how to use the Auth0 to add authorizati
   requirements: [
     'flask 0.11.1',
     'python-jose 1.3.2',
-    'flask-cors 3.0.2'
+    'flask-cors 3.0.2',
+    'Six 1.10.0 and up'
```

Comment:
Six's latest version seems to be 1.10.0 so `and up` is not needed yet. Also, please use a lowercase format (`six` instead of `Six`)